acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
```

Comment:
That, unfortunately, is a price we pay for deeply pinning dependencies: more security, less dynamism. I haven't audited our third-party dependencies for conditional subdependencies. That said, we haven't received any test reports about failures spawned therefrom.